Description: Fix autopkgtest failure on 32-bit architectures
Bug-Debian: https://bugs.debian.org/888414
Author: Graham Inggs <ginggs@debian.org>
Last-Update: 2018-01-26

--- a/Demo/Python3/test_assert.hxx
+++ b/Demo/Python3/test_assert.hxx
@@ -54,6 +54,11 @@
     test_assert_scaler( description, "int", benchmark, value );
 }
 
+static void test_assert( const char *description, long benchmark, long value )
+{
+    test_assert_scaler( description, "long", benchmark, value );
+}
+
 #ifdef HAVE_LONG_LONG
 static void test_assert( const char *description, long long benchmark, long long value )
 {
